### ISSUE TITLE
test / validation test set

### ISSUE THREAD

**Issue by AXLMRIN** [emilienschultz/activetigger#359]
make sure to tell the test set (not seen by the model during the training) and the validation set (seen during the training) in the different pages (project page when creating it, training page and so on...)

**Comment by emilienschultz** [comment]
ok, done
